spark-bigquery-dsv2/spark-bigquery-dsv2-parent/pom.xml

```diff
@@ -37,6 +37,8 @@
     <dependency>
       <groupId>io.openlineage</groupId>
       <artifactId>openlineage-spark_${scala.binary.version}</artifactId>
+      <version>1.19.0</version>
```

Comment:
The version should only appear in the spark-bigquery-parent POM file, as part of the dependencyManagement

Reply:
Done